reversed_func.py

```diff
@@ -0,0 +1,21 @@
+def reversed_func(text: str) -> str:
+
+    reversed_text = []
+    letter_list = [letter for word in reversed(text.split()) for letter in word if letter.isalpha()]
+
+    for letter in text:
+        if letter.isalpha():
+            reversed_text.append(letter_list.pop())
+        else:
+            reversed_text.append(letter)
+
+    return ''.join(reversed_text)
+
+cases = [
```

Comment:
You are missing if name == main: part

Reply:
That's again my carelessness, I fixed it.